package.json

```diff
@@ -141,7 +141,7 @@
     "@ant-design/icons": "4.x",
     "antd": "4.x",
     "ol": "6.x",
-    "react": "16.x",
-    "react-dom": "16.x"
+    "react": ">=16.x",
+    "react-dom": ">=16.x"
```

Comment:
Do we need to set version 17 as required dependency?

Reply:
Nope. The goal is, that it runs with react 16 **and** 17.